x/ibc/light-clients/solomachine/types/solomachine.go

```diff
@@ -28,3 +32,13 @@ func (h Header) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
 func (hd HeaderData) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
 	return unpacker.UnpackAny(hd.NewPubKey, new(crypto.PubKey))
 }
+
+// UnpackInterfaces implements the UnpackInterfaceMessages.UnpackInterfaces method
+func (csd ClientStateData) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
```

Comment:
we should do a sweep of all proto structs with `Any` to ensure these funcs are there. They are hard to debug we you run into the error